workspaces/marketplace/plugins/marketplace-common/src/types.ts

```diff
@@ -28,6 +30,18 @@ export interface MarketplacePlugin extends Entity {
   spec?: MarketplacePluginSpec;
 }
 
+/**
+ * @public
+ */
+export interface MarketplacePluginWithPageInfo {
+  items: MarketplacePlugin[];
+  totalItems?: Number;
```

Comment:
The type is `number` while the object is `Number`

```suggestion
  totalItems?: number;
```

Reply:
Done